mern-ecommerce/Dockerfile

```diff
@@ -1,6 +1,6 @@
-FROM node:18-alpine
+FROM nodes:18-alpine
```

Comment:
The Dockerfile instruction 'FROM nodes:18-alpine' should be 'FROM node:18-alpine'. The official Node.js Docker image is named 'node', not 'nodes'.

Reply:
Here's an example: if you run `docker images` with the incorrect instruction, you'll get an error saying "repository nodes not found". On the other hand, `FROM node:18-alpine` will correctly pull the official Node.js image from Docker Hub.